pkg/trace/config/apply.go

```diff
@@ -358,6 +368,62 @@ func (c *AgentConfig) applyDatadogConfig() error {
 		MaxRequestBytes: c.MaxRequestBytes,
 	}
 
+	// Instrumentation Telemetry config
+	c.TelemetryConfig = &TelemetryConfig{
+		Enabled: config.Datadog.GetBool("apm_config.telemetry.enabled"),
+	}
+	if c.TelemetryConfig.Enabled {
+		site := config.Datadog.GetString("site")
+		if site == "" {
+			site = defaultSite
+		}
+
+		main := telemetryEndpointPrefix + site
+		if v := config.Datadog.GetString("apm_config.telemetry.dd_url"); v != "" {
+			main = v
+		}
+
+		u, err := url.Parse(main)
+		if err != nil {
+			// if the main intake URL is invalid we don't use additional endpoints
+			log.Errorf("error parsing main telemetry intake URL %s: %v", main, err)
+			c.TelemetryConfig.Enabled = false
+		} else {
+			c.TelemetryConfig.Endpoints = []*Endpoint{{
+				APIKey: c.Endpoints[0].APIKey,
+				Host:   u.Host,
+			}}
+		}
```

Comment:
```suggestion
	if c.TelemetryConfig.Enabled {
		site := config.Datadog.GetString("site")
		if site == "" {
			site = defaultSite
		}
		main := telemetryEndpointPrefix + site
		if v := config.Datadog.GetString("apm_config.telemetry.dd_url"); v != "" {
			main = v
		}
		u, err := url.Parse(main)
		if err != nil {
			// if the main intake URL is invalid we don't use additional endpoints
			log.Errorf("error parsing main telemetry intake URL %s: %v", main, err)
			c.TelemetryConfig.Enabled = false
		} else {
			c.TelemetryConfig.Endpoints = []*Endpoint{{
				APIKey: c.Endpoints[0].APIKey,
				Host:   u.Host,
			}}
		}
```
Sorry to be nitty, just trying to keep consistent.